DFE-Digital/curriculum-materials
# Feature/previewing resources

### Context
So teachers can preview various resource types in line in the browser, and so we don't have to complicate our deploy by depending on libreoffice at runtime, we want to generate previews at the time of content consumption and attach them to our resource model.

### Changes proposed in this pull request
Adds a new resource model which has a file (part of the lesson_bundle download) and a preview.
Update the view to link to the preview files.
Updates the seeds to include previews.

### Guidance to review
Code should look sensible and work.
This work has already been reviewed going into the feature branch, just make sure I've not broken anything rebasing!
Manual testing:
Flatten the database and reseed the app either by the api or the models. Expect everything to work as before.
If you want to test out the preview functionality, generate a preview for one of the resources for one of the seeds eg `unoconv -f png slides.odp` and make sure it's named correctly `mv slides.png slides.preview.png` and then reseed the app. (note I'm going to change the allowed preview type for slide decks to pdf but I'll do that in another pr where I add previews for all the resources)
